Tests/test_PDB_PSEA.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2022 by Valentin Vareskic (valentin.vareskic@gmail.com).
+# All rights reserved. This code is part of the Biopython distribution
+# and governed by its license.  Please see the LICENSE file that should
+# have been included as part of this package.
+
+"""Tests for PDB PSEA."""
+
+import unittest
+from Bio.PDB.PSEA import run_psea, psea, psea2HEC, PSEA
+from Bio.PDB import PDBParser
+from subprocess import getoutput
+from Bio import MissingExternalDependencyError
+import os
+
+os.environ["LANG"] = "C"
+
+if "command not found" or "'psea' is not recognized" in getoutput("psea -h"):
```

Comment:
I would separate this into two lines for better readability and invert the if statement to test for the expected output. If I have my terminal language in e.g. french, it won't say 'command not found'.

e.g.
```
cmd_output = getoutput("psea -h")
if not cmd_output.startswith(...):
    raise MissingExternalDependencyError(...)
```